packages/charts/src/utils/themes/dark_theme.ts

```diff
@@ -395,4 +395,13 @@ export const DARK_THEME: Theme = {
       },
     },
   },
+  metric: {
+    text: {
+      lightColor: '#E0E5EE',
+      darkColor: '#343741',
+    },
+    barBg: '#343741',
+    background: '#1D1E23',
```

Comment:
nit - take it or leave it 😆 

```suggestion
    barBackground: '#343741',
    background: '#1D1E23',
```

Reply:
done in e538ac0cf3b2c42c86795388eec2c9c0a2bcd9e8